components/camel-milo/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>components</artifactId>
+        <version>2.19.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-milo</artifactId>
+    <packaging>jar</packaging>
+    <name>Camel :: Milo</name>
+    <description>Camel OPC UA support</description>
+
+    <properties>
+        <camel.osgi.export.pkg>
+            !*.internal.*,
+            org.apache.camel.component.milo.*
+        </camel.osgi.export.pkg>
+        <camel.osgi.import.before.defaults>
+            com.google.common.*;version="19.0.0"
```

Comment:
That should actually resolve to "bigger than 19.0.0" but there is no upper limit for this as google does not do semantic versioning as OSGi does. Be default this would generate a package import of "bigger than 19.0.0 but smaller than 20.0.0" And this would resolve to exactly on version of guava.